clients/admin-ui/cypress/e2e/locations.cy.ts

```diff
@@ -28,13 +28,13 @@ describe("Locations", () => {
       cy.getByTestId("picker-card-Europe").within(() => {
         cy.getByTestId("select-all").should("not.be.checked");
         assertIsChecked("European Economic Area", false);
-        assertIsChecked("France", true);
```

Comment:
I updated the fixture to use more realistic data, so these tests had to be updated too